Lib/fontTools/designspaceLib/__init__.py

```diff
@@ -5,6 +5,7 @@
 import logging
 import os
 import posixpath
+import plistlib
 import xml.etree.ElementTree as ET
```

Comment:
we need to try and use the cElementTree module on python2.7 (on python3 it's the default, when available).
Do this instead:

```python
try:
	import xml.etree.cElementTree as ET
except ImportError:
	import xml.etree.ElementTree as ET
```